MultiplayerMod/Multiplayer/State/MultiplayerState.cs

```diff
@@ -0,0 +1,30 @@
+﻿using UnityEngine;
+
+namespace MultiplayerMod.Multiplayer.State;
+
+public static class MultiplayerState {
+
+    private static readonly Core.Logging.Logger log = new(typeof(MultiplayerState));
+
+    private static MultiplayerRole role = MultiplayerRole.None;
+
+    public static MultiplayerRole Role {
+        get => role;
+        set {
+            role = value;
+            log.Trace(() => $"Setting role to {role}");
+        }
+    }
+
+    public static GameObject GameObject { get; set; }
```

Comment:
What does this gameObject mean?

Reply:
I had an idea to have an object all the multiplayer scripts are attached to, in order to destroy them properly in the end of multiplayer session. Now I'm not sure we need one right here so I'll remove it.